### ISSUE TITLE
RealURL and exception: The identity property "title-2016-07-02-08-00" is no UID.

### ISSUE THREAD

**Issue by cundd** [lochmueller/calendarize#81]
We run into an exception when we use calendarize together with RealURL.

The problem occurs when we try to open an Event detail (the overview is working correctly).

```
#1297759968: Exception while property mapping at property path "": The identity property "the-event-title-2016-07-02-08-00" is no UID.
```

The exception is always thrown if we are logged in to the backend.
But also some clients reported that problem occurred.

Some facts:
PHP 7.0.7
TYPO3 7.6.9
RealURL 2.0.14
Calendarize 2.3.0
my_redirects 3.0.0

We use RealURL's automatic configuration feature.

**Comment by lochmueller** [comment]
Hey @cundd 
do you try this patch? https://github.com/lochmueller/calendarize/commit/8be3c69297c3b534d80010053643e4b3182e19ee
Will be part of the next release...

Regards,
Tim

**Comment by cundd** [comment]
Thank you that worked!

Thank you very much for the fast support!

**Comment by lochmueller** [comment]
Great. I published release 2.3.1 a few seconds ago. Have fun with calendarize...
